varats/paper/case_study.py

```diff
@@ -62,17 +62,15 @@ def __repr__(self) -> str:
                     commit_id=self.commit_id)
 
 
-class CSStage(yaml.YAMLObject):
+class CSStage():
     """
     A stage in a case-study, i.e., a collection of revisions. Stages are used
     to separate revisions into groups.
     """
-
-    yaml_loader = yaml.SafeLoader
-    yaml_tag = u'!CSStage'
-
-    def __init__(self, name: tp.Optional[str] = None) -> None:
-        self.__revisions: tp.List[HashIDTuple] = []
+    def __init__(self,
+                 name: tp.Optional[str] = None,
+                 revisions: tp.List[HashIDTuple] = []) -> None:
```

Comment:
```suggestion
                 revisions: tp.Optional[tp.List[HashIDTuple]] = None) -> None:
```
Default parameter in python have reference binding, meaning if you add something to the list in this object other objects will see the same list

Reply:
Resolved in 4cd506d